btc-types/src/header.rs

```diff
@@ -26,4 +26,6 @@ pub struct ExtendedHeader {
     pub chain_work: Work,
     /// Block height in the Bitcoin network
     pub block_height: u64,
+    // The parent block if AuxPow is used (for Dogecoin)
+    pub aux_parent_block: Option<H256>,
```

Comment:
We will not be able to decode the exist btc blocks if we update the contract. Is it needed in the light version? If not then we can remove it from the light

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/101/commits/f454f6a780eb4e2e9836646432cfa00f8d080d04